CMakeLists.txt

```diff
@@ -26,8 +26,13 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+option(CLIENT_SIMULATOR "Install the client simulator binary" OFF)
+if (CLIENT_SIMULATOR)
+    cmake_minimum_required(VERSION 3.1)
```

Comment:
pfClientSimulator is using python3 explicitly.
CMake 2.8 has a known bug concerning finding python when several version are installed on host :
http://www.cmake.org/Bug/print_bug_page.php?bug_id=13794
That's why ...